server/lib/pbench/indexer.py

```diff
@@ -4584,7 +4654,7 @@ def mk_result_data_actions(self):
                 else:
                     # Only the parent result data documents hold the tracking IDs.
                     source["@generated-by"] = self.idxctx.get_tracking_id()
-                count += 0
+                count += 1
```

Comment:
@ndokos, doh!  Can't believe we missed this!

Reply:
It wasn't near the lamp post :-) - but I saw the changes in all the gold files and was wondering.